test/elixir_search_extractor/account/oauth2/auth_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule ElixirSearchExtractor.Account.Oauth2.AuthTest do
+  use ElixirSearchExtractor.DataCase, async: true
+
+  import ElixirSearchExtractor.AccountsFixtures
+  alias ElixirSearchExtractor.Account.Oauth2.Auth
+
+  describe "authenticate/2" do
+    test "it authenticates user if valid email and password is given" do
```

Comment:
Should we remove `it` 🤔 

```suggestion
    test "authenticates user if valid email and password is given" do
```

Same for other tests

Reply:
@rosle removed in [d0ba2b0](https://github.com/rafayet-monon/elixir_search_extractor/pull/52/commits/d0ba2b0a7e22a8c754c917f877f88ee2f979c09e)